src/components/ec-file-list/ec-file-list.vue

```diff
@@ -0,0 +1,111 @@
+<template>
+  <div
+    data-test="ec-file-list"
+    class="ec-file-list"
+  >
+    <div
+      v-for="(item, index) in items"
+      :key="index"
+      class="ec-file-list__item"
+      :data-test="`ec-file-list__item ec-file-list__item--${index}`"
+    >
+
+      <ec-icon
+        class="ec-file-list__document-icon"
+        data-test="ec-file-list__document-icon"
+        name="simple-document"
+        type="interactive"
+        :size="20"
+      />
+
+      <span
+        class="ec-file-list__name"
+        :title="item.name"
+        :data-test="`ec-file-list__name ec-file-list__name--${index}`"
+      >
+        {{ item.name }}
+      </span>
+
+      <ec-btn
+        :is-disabled="isDeleteDisabled"
+        :is-submit="false"
+        :icon="'simple-outline-delete'"
+        class="ec-file-list__delete-btn"
+        :data-test="`ec-file-list__delete-btn--${index}`"
+        @click="onDelete(item)"
+      >
+        <span class="tw-sr-only">Delete document</span>
+      </ec-btn>
+    </div>
+  </div>
+</template>
+
+<script>
+import EcBtn from '../ec-btn';
+import EcIcon from '../ec-icon';
+
+export default {
+  name: 'EcFileList',
+  components: {
+    EcIcon,
+    EcBtn,
+  },
+  props: {
+    items: {
+      type: Array,
+      default: () => [],
+    },
+    isDeleteDisabled: {
+      type: Boolean,
+      default: false,
+    },
+  },
+  methods: {
+    onDelete(item) {
+      this.$emit('delete', item);
+    },
+  },
+};
+</script>
+
+<style>
+  .ec-file-list {
+    &__item {
+      @apply tw-flex tw-flex-row tw-items-center;
+    }
+
+    &__name {
+      @apply tw-w-full;
+      @apply tw-px-4;
+      @apply tw-text-gray-3;
+      @apply tw-truncate;
+    }
+
+    &__document-icon {
+      @apply tw-w-24 tw-h-24;
+      @apply tw-fill-gray-4;
+    }
+
+    &__delete-btn {
+      @apply tw-border-none;
+      @apply tw-bg-transparent tw-fill-gray-4;
+      @apply tw-pr-0;
+
+      @media screen and (max-width: 480px) {
```

Comment:
We try to develop mobile first, so we should have code for smaller screens without `@media` and write `media` queries for bigger screens. So this could be written as 
```
@apply tw-px-0;

@media screen and (min-width: 480px) {
  @apply tw-px-....;
}
```

Ideally, we should first try to re-use one of the predefined breakpoints: https://github.com/Ebury/chameleon/blob/master/tailwind.config.js#L17. If those doesn't work, we can use a custom breakpoint like `480px`.

Tailwind has a nice helper called `@screen` to make it even easier to re-use existing breakpoints, e.g. from ec-donut.vue
```
    @apply tw-mr-0;
    @apply tw-flex tw-justify-center;

    @screen sm {
      @apply tw-w-auto;
      @apply tw-mr-24;
      @apply tw-block;
      @apply tw-justify-start;
    }
```

will generate `@media (min-width: 576px)` in the final CSS

Reply:
We're basically following this guide: https://tailwindcss.com/docs/responsive-design